testData/ide/formatter/simpleOneFormatted.rho

```diff
@@ -0,0 +1,7 @@
+new time_contract in {
```

Comment:
The test is failing for this file, actual file has 4 spaces.

Reply:
I have fixed the text file. 
Two or four spaces should be set by users. Wait for my further PR. : )